docs/src/paradox/07-sipi/setup-sipi-for-knora.md

```diff
@@ -23,16 +23,22 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
 
 In order to serve files to the client application like the Salsah GUI,
 Sipi must be set up and running. Sipi can be downloaded from its own
-GitHub repository: <https://github.com/dhlab-basel/Sipi>. Please follow
-the instructions given in the README to compile it on your system.
+GitHub repository: <https://github.com/dhlab-basel/Sipi> (which requires
+building from source), or the published [docker image](https://hub.docker.com/r/dhlabbasel/sipi/).
+can be used. To start Sipi, run the following command from inside the `sipi/`
+folder:
 
-Once it is compiled, you can run Sipi with the following option:
-`./local/bin/sipi --config=config/sipi.knora-config.lua` (or
-`./local/bin/sipi --config=sipi.knora-test-config.lua` for using sipi
-for testing). Please see `sipi.knora-config.lua` for the settings like
-URL, port number etc. These settings need to be set accordingly in
-Knora's `application.conf`. If you use the default settings both in Sipi
-and Knora, there is no need to change these settings.
+```
+$ export DOCKERHOST=LOCAL_IP_ADDRESS
+$ docker run --rm -it --add-host webapihost:$DOCKERHOST -v $PWD/config:/sipi/config -v $PWD/scripts:/sipi/scripts -v /tmp:/tmp -v $HOME:$HOME -p 1024:1024 dhlabbasel/sipi:develop /sipi/local/bin/sipi --config=/sipi/config/sipi.knora-docker-config.lua
```

Comment:
Is it normal that Sipi prints every log message twice?

```
$ docker run --rm -it --add-host webapihost:$DOCKERHOST -v $PWD/config:/sipi/config -v $PWD/scripts:/sipi/scripts -v /tmp:/tmp -v $HOME:$HOME -p 1024:1024 dhlabbasel/sipi:develop /sipi/local/bin/sipi --config=/sipi/config/sipi.knora-docker-config.lua

Sipi Version v1.2.0 Beta (build 2018-04-05 11:49)
Sipi Source Git-Version: 454d1d1
Subdir migration of /sipi/images/knora....
act_levels=0 levels=1
Sipi: Sipi Version v1.2.0 Beta (build 2018-04-05 11:49)
Sipi: Sipi Version v1.2.0 Beta (build 2018-04-05 11:49)
Sipi: Sipi Source Git-Version: 454d1d1
Sipi: Sipi Source Git-Version: 454d1d1
Sipi: Cache at "/sipi/cache" cachesize=104857600 nfiles=0 hysteresis=0.100000
Sipi: Cache at "/sipi/cache" cachesize=104857600 nfiles=0 hysteresis=0.100000
Sipi: Sipi server starting
Sipi: Sipi server starting
Sipi: Serving images from /sipi/images
Sipi: Serving images from /sipi/images
Sipi: Starting shttps server with 8 threads
Sipi: Starting shttps server with 8 threads
Sipi: Added route /make_thumbnail with script /sipi/scripts/make_thumbnail.lua
Sipi: Added route /make_thumbnail with script /sipi/scripts/make_thumbnail.lua
Sipi: Added route /convert_from_binaries with script /sipi/scripts/convert_from_binaries.lua
Sipi: Added route /convert_from_binaries with script /sipi/scripts/convert_from_binaries.lua
Sipi: Added route /convert_from_file with script /sipi/scripts/convert_from_file.lua
Sipi: Added route /convert_from_file with script /sipi/scripts/convert_from_file.lua
Sipi: Added route /Knora_login with script /sipi/scripts/Knora_login.lua
Sipi: Added route /Knora_login with script /sipi/scripts/Knora_login.lua
Sipi: Added route /Knora_logout with script /sipi/scripts/Knora_logout.lua
Sipi: Added route /Knora_logout with script /sipi/scripts/Knora_logout.lua
Sipi: Server listening on port 1024
Sipi: Server listening on port 1024
```

Reply:
Yes, because the way logging is set up in sipi.

Reply:
release notes updated